digital_benchmark/instagram_benchmark/views.py

```diff
@@ -120,6 +120,36 @@ def _load_media(self, dataProvider, instagram_parser, current_user, access_token
         instagram_parser.save_media_insight_data(
             all_user_media, current_user, access_token)
 
+class StartCrawlerView(APIView):
+   permission_classes = [IsAuthenticated]
+ 
+   def post(self, request):
+       app_user_id = request.user.id
+       unique_id = str(uuid4())
+       public_username = request.data.get('username')
+       setting = {
```

Comment:
function "_trigger_crawler()" is already receiving "unique_id". We can make "setting" dict inside the function "_trigger_crawler" and make the view clean.